sdk/search/azure-search-documents/swagger/src/main/java/SearchIndexCustomizations.java

```diff
@@ -16,20 +19,15 @@
     private static final String IMPLEMENTATION_MODELS = "com.azure.search.documents.implementation.models";
     private static final String MODELS = "com.azure.search.documents.models";
 
-    // Classes
-    private static final String SEARCH_OPTIONS = "SearchOptions";
-    private static final String AUTOCOMPLETE_OPTIONS = "AutocompleteOptions";
-    private static final String SUGGEST_OPTIONS = "SuggestOptions";
-
     @Override
     public void customize(LibraryCustomization libraryCustomization, Logger logger) {
         customizeModelsPackage(libraryCustomization.getPackage(MODELS));
         customizeImplementationModelsPackage(libraryCustomization.getPackage(IMPLEMENTATION_MODELS));
     }
 
     private void customizeModelsPackage(PackageCustomization packageCustomization) {
-        customizeAutocompleteOptions(packageCustomization.getClass(AUTOCOMPLETE_OPTIONS));
-        customizeSuggestOptions(packageCustomization.getClass(SUGGEST_OPTIONS));
+        customizeAutocompleteOptions(packageCustomization.getClass("AutocompleteOptions"));
```

Comment:
is that better to keep using the static final String instead of creating String object each time?

Reply:
This class should only be called once during code generation so it is effectively the same. I switched to this pattern to make it clearer on what is being changed.